pallets/phala/src/mining.rs

```diff
@@ -635,7 +649,10 @@ pub mod pallet {
 						if !miner_info.state.is_mining() {
 							continue;
 						}
-						miner_info.state = MinerState::MiningUnresponsive;
+						// Gk should not change state of a miner in coolingdown
+						if miner_info.state != MinerState::MiningCoolingDown {
```

Comment:
According to the `fn is_mining()`:
```
		fn is_mining(&self) -> bool {
			matches!(
				self,
				MinerState::MiningIdle | MinerState::MiningUnresponsive
			)
		}
```
Looks like it would not run to here if the `miner_info.state` is `MinerState::MiningCoolingDown`.